app/api/config.ts

```diff
@@ -0,0 +1,38 @@
+import { Tenant } from './tenants/tenantContext';
+
+const { ROOT_PATH, UPLOADS_FOLDER, CUSTOM_UPLOADS_FOLDER } = process.env;
+
+const rootPath = ROOT_PATH || `${__dirname}/../../`;
+
+// this needs to be true in order for uwazi to work properly
+// when using multiple node processes
+const CLUSTER_MODE = process.env.CLUSTER_MODE || false;
+
+export const config = {
+  PORT: process.env.PORT || 3000,
+
+  DBHOST: process.env.DBHOST ? `mongodb://${process.env.DBHOST}/` : 'mongodb://localhost/',
+
+  rootPath,
+
+  publicAssets: `${rootPath}/public/`,
+
+  // db for tenants list and sessions
+  SHARED_DB: 'uwazi_shared_db',
+
+  multiTenant: process.env.MULTI_TENANT || false,
+  clusterMode: CLUSTER_MODE,
+  defaultTenant: <Tenant>{
+    dbName: process.env.DATABASE_NAME || 'uwazi_development',
+    indexName: process.env.INDEX_NAME || 'uwazi_development',
+    uploadedDocuments: UPLOADS_FOLDER || `${rootPath}/uploaded_documents/`,
+    attachments: UPLOADS_FOLDER || `${rootPath}/uploaded_documents/`,
+    customUploads: CUSTOM_UPLOADS_FOLDER || `${rootPath}/custom_uploads/`,
```

Comment:
@daneryl the default tenant is missing the temporalFiles path right?

Reply:
yes it was missing, probably missed it on the rebase of the export, good catch 👍 .
it is fixed now.